cypress/README.md

```diff
@@ -1,141 +1,48 @@
-3. Locators should be named:
-  * Component on preview - `componentPreview`;
-  * Knobs - `componentKnobs`.
+Details of how to write Cypress tests, including coding standards and Applitools integration, can be found in the [Carbon Testing Styleguide](https://github.com/Sage/carbon/blob/master/docs/testing-styleguide.md).
 
 ## Continuous Integration (CI)
-Every commit / pull request in repository initializing cypress tests and runs:
-### Travis
-1. `npm run lint` - runs lint test.
-2. `npm test` - runs jest tests.
-3. `npm start` - runs storybook.
-4. `wait-on http://localhost:9001` - waits until storybook is up and running and is ready to run tests.
-5. `npm run cypress:ci:visual` - run all `visual` test suite.
-### GitHub Actions
-1. `npm start` - runs storybook.
-3. `wait-on http://localhost:9001` - waits until storybook is up and running and is ready to run tests.
-4. `npx cypress run --parallel --record` - run all tests suite.
-  * NOTE: if the tests failed you need to make another `commit`/`push` to generate `cypress run_id` to make possible to re-run cypress tests (on each run `run_id` should be unique).
+Every commit/pull request in the repository initiates Cypress test runs. Visual regression tests run in Travis. All other regression tests run in GitHub Actions.
 
-## Dashboards
+### Travis
+1.	`npm start` - runs Storybook.
+2.	`wait-on http://localhost:9001` - waits until Storybook is up and running and is ready to run tests.
+3.	`npm run cypress:ci:visual` - runs the complete visual test suite.
+4.	The build result can be seen in GitHub in the pull request/branch and the detailed results can be seen in the [Applitools dashboard](https://eyes.applitools.com/app/test-results/00000251811810944038/?accountId=MZDiTwN5_kOmMbjBqRi9pw~~) linked from the pull request/branch checks.
 
-### Cypress dashboard
-[Cypress.io - dashboard](https://dashboard.cypress.io/projects/8458bb/runs)
+### GitHub Actions
+1.	`npm start` - runs storybook.
+2.	`wait-on http://localhost:9001` - waits until storybook is up and running and is ready to run tests.
+3.	`npx cypress run --parallel –record` - runs the complete test suite..
+4.	The build result can be seen in GitHub in the pull request/branch and the detailed results can be seen in the [Cypress.io dashboard](https://dashboard.cypress.io/projects/8458bb/runs) or in GitHub Actions, both linked from the pull request/branch checks.
 
-### Applitools dashboard
-[Applitools - dashboard](https://eyes.applitools.com/app/test-results/00000251812391553929/?accountId=MZDiTwN5_kOmMbjBqRi9pw~~)
\ No newline at end of file
+  NOTE: If the tests failed for any reason such as if there is an issue with GitHub or the Cypress dashboard and we need to re-run the run exactly as it was, you need to make another commit/push to generate a cypress run_id to re-run the Cypress tests (for each run the run_id should be unique). Use `git commit –amend` then `git push --force-with-lease` to push and trigger a new build.
```

Comment:
it's `git commit –-amend`
Ie. a double hyphen.

Reply:
This process has since been amended so I've updated the whole NOTE bit with the changes.